### ISSUE TITLE
OTPP ADE "Important” icons don’t pass color contrast check*

### ISSUE THREAD

**Issue by jtmst** [department-of-veterans-affairs/va.gov-team#120021]
**Issue: “Important” icons don’t pass color contrast check**

* **Page:** Current copay balances, Current debts  
* **First observed:** MacOS \+ Chrome  
* **Expected 508 Office score:** Critical  
* **Expected CC severity level:** 1  
* **Expected CC experience standard:** User can perceive all meaningful elements  
* **Expected WCAG violation:** 1.4.3 Contrast  
* **Description:** These icons use a color that doesn’t pass WCAG contrast standards:  
   <img width="461" height="39" alt="Image" src="https://github.com/user-attachments/assets/48dcc46f-3d1e-4b87-aeef-30c0cb012baf" />
* **How to recreate:** Use a color contrast checker (I used [CCA](https://www.tpgi.com/color-contrast-checker/)) to check the contrast of the icons  
* **Recommendation:** Use \-–vads-color-orange instead. This passes contrast rules. It’s also what’s used on the VADS website:   
  <img width="180" height="39" alt="Image" src="https://github.com/user-attachments/assets/863f90d9-e259-4ea8-b975-69541331d47b" />

_Provide a technical description of the task._

- Given: (The beginning state of the scenario)      
- When: (Specific action that the user takes)      
- Then: (Testable outcome that can be captured in the testing section, usually caused by the action in When)              

## Tasks
- [ ] Task 1
- [ ] Task 2
- [ ] Task 3          

## Design and Front End sync on implementation of design
- [ ] Required
- [ ] Complete
- [ ] Not Required           

## Acceptance Criteria
- [ ] Item one from the description performs as described in the description
- [ ] Item two from the description performs as described in the description      

## Testing
- [ ] N/A if non development work
- [ ] Testing passed and documented in this ticket based off the "Then" statement in the description     

Test procedures:      
*If additional testing steps or credentials to perform testing are needed list them here*    

## Bug Ticket Resolution  
- [ ] If this is a Bug or Fix - Check this box and complete the information below.
- Additional description of bug once the cause was identified.  
      - Answer
- Root Cause - what caused this issue?  
      - Answer
- Impact - how were Veterans impacted? (i.e. FSR submissions which ones, date range, etc)  
      - Answer
- Resolution Details - what was the fix & when did it go live.  
      - Answer
- Follow Up Actions - any additional/further work that came out of fixing the bug?  
      - Answer
- Preventative Measures - any planned steps to prevent similar issues in the future?  
      - Answer

</details>
